ILSpy/TreeNodes/AssemblyReferenceTreeNode.cs

```diff
@@ -45,7 +46,10 @@ public AssemblyReferenceTreeNode(AssemblyReference r, AssemblyTreeNode parentAss
 			get { return Language.EscapeName(r.Name) + GetSuffixString(r.Handle); }
 		}
 
-		public override object Icon => Images.Assembly;
+		public override object Icon => moduleResolveFailed switch {
```

Comment:
I know switch is the hotness, but was is wrong with boolVar ? trueResult : falseResult ...

Reply:
`moduleResolveFailed` is nullable bool, I use switch in preparation for the future. We might think about adding another icon when the AssemblyReference information hasn't been loaded yet.